### ISSUE TITLE
Todo:

### ISSUE THREAD

**Issue by xclusive36** [xclusive36/SnipSnap#40]
These are the tasks that need to be completed for a minimum viable product (mvp) before the project can be submitted.
If you have any ideas on things that can be added to the project, please create a new issue and self assign before starting work.

MVP:
- [x] Home page - Show working appointment form to logged in users. Otherwise disable form for non logged in users.
- [x] Home page - Complete appointment form so logged in users may book appointments.
- [x] Login page - Convert HTML in Login page to Ionic. Form elements should be converted to IonInput, etc..
- [x] Signup page - Convert HTML in Signup page to Ionic. Form elements should be converted to IonInput, etc..
- [x] Profile page - Convert HTML in Profile page to Ionic. The Profile page should pull appointment data from server and display to user as a list.

Extra:

- [x] Profile page - The ability to remove appointments that have been already booked with a prompt to make sure the user wishes to do so.
- [x] Profile page - Add a Toast (notification popup) when appointment has been removed notifying the user that the removal has been successfully completed.
- [x] Home page - Add a Toast when an appointment has been booked. notifying the user that the booking has been successfully completed.
- [ ] Home page - Move booking form to it's own component for reusability.
- [ ] Profile page - use new booking component (1 line above) to give user ability to modify their appointment.
- [x] Visual - Anything that can be visually improved upon (collors, etc..)
- [x] ???

**Comment by xclusive36** [comment]
- [ ] Home page - Add 'total cost' and display total cost of service selected at bottom of appointment form.
- [x] Home page - Add prompt to confirm selection choice before booking appointment.
- [x] Home page - We should create a logo
- [ ] New page - We may want to consider moving the booking form to it's own page.

**Comment by xclusive36** [comment]
- [x] New page - We may want to add a Contact us page. 
- [x] Footer - We may want to add Store Hours and set the booking appointments time to only accept within those hours.

**Comment by xclusive36** [comment]
Also we may want to add an about section or page. If booking gets moved to it's own page, 'about' can be on the home page.
